Header header
# Array of Nash equilibrium points for each robot
# The index in the array corresponds to (robot_id - 1)
geometry_msgs/Point[] nash_points
